library/alloc/tests/vec.rs

```diff
@@ -518,6 +518,22 @@ fn test_swap_remove_empty() {
     vec.swap_remove(0);
 }
 
+#[test]
+fn test_try_remove() {
+    #[feature(vec_try_remove)]
```

Comment:
This needs to be at the crate level I think, and changed to `#![feature(...)]`

Reply:
Oh yeah, I just realized